src/java.desktop/share/classes/javax/swing/plaf/synth/SynthTabbedPaneUI.java

```diff
@@ -655,9 +671,17 @@ private void paintTab(SynthContext ss, Graphics g,
             paintIcon(g, tabPlacement, tabIndex, icon, iconRect, isSelected);
             paintText(ss, g, tabPlacement, font, metrics,
                     tabIndex, clippedTitle, textRect, isSelected);
+        }
+    }
 
-
+    private Color getUnselectedBackgroundAt(int index) {
```

Comment:
I guess `getUnselectedBackground` would be right naming convention.

Reply:
Don't think so as there is an `index` argument.